app/renderer/src/components/Inspector/Source.jsx

```diff
@@ -20,27 +21,98 @@ const Source = (props) => {
     methodCallInProgress,
     mjpegScreenshotUrl,
     isSourceRefreshOn,
+    pageSourceSearchText,
     t,
   } = props;
 
+  /**
+   * Highlights the part of the node text in source tree that matches the search term.
+   */
+  const higlightNodeMatchingSearchTerm = (nodeText, searchText) => {
+    if (!searchText) {
+      return nodeText;
+    }
+
+    const index = nodeText.indexOf(searchText);
+    if (index < 0) {
+      return nodeText;
+    }
+    const prefix = nodeText.substring(0, index);
+    const suffix = nodeText.slice(index + searchText.length);
+    //Matched word will be wrapped in a separate span for custom highlighting
+    const matchedWord = nodeText.slice(index, index + searchText.length);
```

Comment:
I assume `matchedWord` is used due to potentially different casing from `searchText`?

Reply:
Yup. Searching is current not case sensitive. So to preserve the highlighted text casing we are using the `matchedWord` to extract the exact word from the source.

Reply:
can we just do `matchedWord = suffix`?